episodes/08-motivation.md

```diff
@@ -61,7 +61,7 @@ things in the opposite corner
 that are time-consuming to learn and have little near-term application
 should be avoided in our workshops.
 
-![](fig/what-to-teach.png){alt='A stylized graph with y-axis labeled "usefulness once mastered" and and x-axis labeled "mean time to master". The upper left quadrant says "teach this first" and the lower right quadrant says "do not bother".'}
+![](fig/what-to-teach.png){alt='A stylized graph with y-axis labeled "usefulness once mastered" and and x-axis labeled "mean time to master". The upper left quadrant says "teach this first" and the lower right quadrant says "don't bother".'}
```

Comment:
```suggestion
![](fig/what-to-teach.png){alt='A stylized graph with y-axis labeled "usefulness once mastered" and and x-axis labeled "mean time to master". The upper left quadrant says "teach this first" and the lower right quadrant says "do not bother".'}
```
contractions are bad for readability, even if this makes a slight mismatch, it is important, I think, for the alt text to represent best practices. we should change the image to be "do not" not make the alt text employ bad practices